.github/workflows/test.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           check-latest: true
           cache: true
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v6
```

Comment:
https://github.com/nikolaiianchuk/go-swagger/pull/1

Updating the github action seems to fix it, see the lint stage